docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
       - GITEA__database__USER=gitea
       - GITEA__security__INTERNAL_TOKEN=${GITEA_INTERNAL_TOKEN}
       - GITEA__session__PROVIDER=db
+      - GITEA__cron__UPDATE_MIRRORS="SCHEDULE = @every 10m"
```

Comment:
I think maybe? 

```suggestion
      - GITEA__cron__UPDATE_MIRRORS__SCHEDULE = "@every 10m"
```

Reply:
Doens't seem to work

![image](https://github.com/kitspace/kitspace-v2/assets/37152329/48c9893e-6c51-4eb2-a554-7941aec1c801)